libuuu/CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ find_package(PkgConfig REQUIRED)
 pkg_check_modules(LIBUSB REQUIRED libusb-1.0>=1.0.16)
 pkg_check_modules(LIBZSTD REQUIRED libzstd)
 find_package(Threads)
+pkg_check_modules(TINYXML2 REQUIRED tinyxml2)
 
```

Comment:
it blocked windows build.  Windows have not use cmake.  Github CI have problem now. I am sure it will be build fail at windows platform.

Reply:
Right. Need a Windows support. Let me try to add it.

Reply:
Added CI workflow dependency for tinyxml2 and Windows support